recipe/meta.yaml

```diff
@@ -1,33 +1,39 @@
 {% set name = "qstylizer" %}
-{% set version = "0.1.10" %}
+{% set version = "0.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/blambright/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: a5d561ef757d1751766674fba977e339cb4028b057a350969224403f286c4751
+  sha256: 95f7666929cc34efbbd9e0640a13f507b8ec35ec34a9604017cf96c24f7f9ea6
 
 build:
-  noarch: python
   number: 0
-
+  skip: true  # [s390x]
+  # inflection is not available for s390x. qstylizer is built as a new dependency for spyder, which we don’t support on linux-s390x. Therefore we’re skipping this platform for this build.
+  skip: true  # [py<37]
```

Comment:
@marcoesters @varlackc 
Can this be done via one line or is it better to have them separate like this?

Reply:
This can be one line, yes.

Some may prefer a separation of python and architecture logic if they are independent from each other.